test/views/CashAccountDeclarationDetailsSearchNoResultSpec.scala

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import utils.SpecBase
+import behaviours.{ComponentDetailsForAssertion, LinkDetails, GuidancePageBehaviour}
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import views.html.cash_account_declaration_details_search_no_result
+
+class CashAccountDeclarationDetailsSearchNoResultSpec extends SpecBase with GuidancePageBehaviour {
+
+  val accNumber = "12345678"
```

Comment:
How come we are not using a Setup trait here?

Reply:
Cause we are using the GuidancePageBehaviour